state/graphql-sender/src/main/java/com/expediagroup/streamplatform/streamregistry/state/graphql/DefaultApolloClientFactory.java

```diff
@@ -34,12 +33,6 @@
   private Credentials credentials;
   @NonNull private final Consumer<OkHttpClient.Builder> configurer;
 
-  public DefaultApolloClientFactory(String streamRegistryUrl) {
```

Comment:
For OSS users, they may still wish to use anonymous so these should remain.

Reply:
done